code/game/machinery/computer/abnormality_work.dm

```diff
@@ -104,7 +104,32 @@
 
 /obj/machinery/computer/abnormality/proc/start_work(mob/living/carbon/human/user, work_type, training = FALSE)
 	var/sanity_result = round(datum_reference.current.fear_level - get_user_level(user))
-	var/sanity_damage = -(max(((user.maxSanity * 0.26) * (sanity_result)), 0))
+	var/sanity_damage
+	if (istype(datum_reference.current, /mob/living/simple_animal/hostile/abnormality/white_night)) // I wanna check for WN, and if so give him a custom work fear rating. Ideally I'd get a console subtype but I'm unsure how to properly pre-set it to be a WN console for WN.
```

Comment:
I feel like this and CENSORED checks should be generalized.

Reply:
Best way I can think of would be setting a list of "reactions" to the abno based on a list, I.e. 1 level difference is X level fear reaction, etc. and having WN and CENSORED overwrite that

Reply:
and I've realized that would require overwriting it for each threat level... hmm. I feel like that might just be less efficient than this.